### ISSUE TITLE
Ozone content metadata and two thumbnails

### ISSUE THREAD

**Issue by i30817** [libretro/RetroArch#14634]
## Description

With three-two (if the thumbnails are the 'main' ones selected) thumbnails content metadata is not displayed at all. However if the game happens to only have 1 thumbnail available to display in the current mode, it will.

This is probably because there is no space. 

### Expected behavior

What i'd _like_ to happen is a ticker variant that is really a single line and display all 3 thumbnails at once in the column if they exist, and the ticker variant of content metadata being pinned to the bottom. I'd have to see how it looks but it seems like it should be possible from the space if the ticker was really a single line (it isn't).

### Actual behavior

It's somewhat inconsistent. It's better than what it was before, because at least when you press the button to switch thumbnails there are no '3' thumbnail states like before, but only two, and that's good, but it would be interesting to see a state that displayed all the thumbnails and the content metadata. Failing that, at least when there are two visible ones, not hide the content metadata, maybe forcing the 'ticker' state.

Another, easier, way to achieve consistency would be a option in the appearance menu to always hide the content metadata.

@sonninnos

**Comment by sonninnos** [comment]
Sure, and also finding a proper solution for playlist items that don't have a boxart but do have screenshots, like C64. It kinda works only when second thumbnail is enabled, but with only the main one being boxart, it shows nothing. It looked kinda messy to know beforehand if the image really exists or not instead of being simply enabled.

**Comment by i30817** [comment]
Yes, that's another thing i noticed. It makes one paranoid that the thumbnails are either missing or not downloading to the right name properly (in my case, this last one is because i actually have a custom thumbnail downloader i programmed myself so it made me wonder if i was removing the forbidden characters correctly - i was).

**Comment by i30817** [comment]
Also if you select the ticker option, the metadata text doesn't display empty lines between the actual text, unlike the normal display. This seems inconsistent, and i assume unintentional. To display the text with both images, it might be sufficient to remove the extra \n?
